debian/changelog

```diff
@@ -0,0 +1,5 @@
+gtsam (4.0.0-1berndpfrommer) bionic; urgency=medium
```

Comment:
change name?

Reply:
Ooops.... Sorry I missed notifications and forgot about this PR :-) 

> change name?

I wouldn't change the name there, since we are actually adding new versions on top of his former release :-)
Now, new entries in the change log should have the name of whoever actually maintains the Debian packaging, for sure, which might be different than those maintaining the upstream/original project.